datadog_checks_dev/datadog_checks/dev/tooling/commands/env/check.py

```diff
@@ -14,8 +14,23 @@
 )
 @click.argument('check')
 @click.argument('env', required=False)
-@click.option('--rate', '-r', is_flag=True)
-def check_run(check, env, rate):
+@click.option(
+    '--rate', '-r', is_flag=True, help='Compute rates by running the check twice with a pause between each run'
+)
+@click.option('--times', '-t', type=click.INT, help='Number of times to run the check')
```

Comment:
Any reason to not call it --check-times like the agent does?

Reply:
I like shorter names, we do this above too with `rate`